1-exercises/E-for-loop/exercise2.js

```diff
@@ -27,7 +27,10 @@ const AGES = [
 ];
 
 // TODO - Write for loop code here
-
+for (i = 0; i < WRITERS.length; i++) {
+  let text = WRITERS[i] + " is " + AGES[i] + " years old";
```

Comment:
Great! Also have a look at [string templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), see if you prefer that syntax.